frontend/src/components/common/Notification.js

```diff
@@ -0,0 +1,54 @@
+import { alert, Stack, defaultModules } from "@pnotify/core";
+import * as PNotifyMobile from "@pnotify/mobile";
+import _ from "lodash-es";
+
+// Set up PNotify with mobile module
+defaultModules.set(PNotifyMobile, {});
+
+// Create a notification stack
```

Comment:
Lets also try to add react component, i think we might want for every PR added, we should include react in our Pr.